azuredevops/internal/service/taskagent/resource_variable_group.go

```diff
@@ -554,3 +648,11 @@ func flattenAllowAccess(d *schema.ResourceData, definitionResource *[]build.Defi
 	}
 	d.Set(vgAllowAccess, allowAccess)
 }
+
+func getSecretName(secretID string) (secret string) {
+	if len(secretID) == 0 {
+		return ""
+	}
+	secretURL := strings.Split(secretID, "/")
+	return secretURL[len(secretURL)-1]
```

Comment:
I'm a bit confused, what's happening here?

Reply:
Azure key secret ID returned in http format. The real secret name is at the end of the ID.
Service response:
```json
{
            "contentType": "1222",
            "id": "https://xzadotest.vault.azure.net/secrets/secret1",
            "attributes": {
                "enabled": true,
                "exp": 1675424159,
                "created": 1611734011,
                "updated": 1612353657,
                "recoveryLevel": "Recoverable+Purgeable"
            },
            "tags": {}
        },
```